cypress/support/MonitorDocumentChanges.js

```diff
@@ -121,8 +136,8 @@ var MonitorDocumentChanges = (function()
         if(currentHtml != lastHtml)
         {
             helpers.ConLog(thisFuncName, `Change Found - Milliseconds since last change: ${(currentTime - lastChangeTime)}`)
-            //cy.writeFile(currentHtml, `c:\\temp\\dom.${helpers.NowAsString()}.txt`)
-            if (dumpHtml || expectingSpinner) helpers.ConLog(thisFuncName, `Current HTML:\n${currentHtml}`)
+            if (dumpHtml || expectingSpinner) 
```

Comment:
Generally prefer all if statements to be wrapped in brackets (i.e. { }), even if only one statements.  Leads to fewer bugs in the long term. Thx